core/lib/libtomcrypt/src/mpi_desc.c

```diff
@@ -0,0 +1,738 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#include <crypto/crypto.h>
+#include <kernel/panic.h>
+#include <mbedtls/bignum.h>
+#include <stdlib.h>
+#include <string.h>
+#include "tomcrypt.h"
+#include "tomcrypt_mp.h"
+#include <util.h>
+#include <mempool.h>
+
+#define MPI_MEMPOOL_SIZE	(42 * 1024)
```

Comment:
Why 42kB? I think libpma is aournd 12kB for 50 numbers of 4096bits.
I agree an oversized value won't hurt pager constraints. 
Yet, maybe a comment stating why this size could be nice.

Reply:
The libmpa size is 52200 bytes
I'll add a comment on the size.